Superlime.py

```diff
@@ -37,13 +38,15 @@ def on_activated(self, view):
 						.replace("UTF-8 ", "utf_8") \
 						.replace("with BOM", "_sig") \
 						.lower()
+			'''
+			encoding = sublime_lib.encodings.from_sublime(encoding)
 			try:
 				encoded = content.replace('\n', self.lineEndings[view.line_endings()]).encode(encoding) if encoding != "Hexadecimal" else binascii.unhexlify(string.replace(" ",""))
 				tempFile.write(encoded)
 			finally:
 				tempFile.close()
 
-			#print view.file_name(), ": Saved temp file with encoding " + encoding
+			print (f"{view.file_name()}: Saved temp file with encoding {encoding}")
```

Comment:
The switch to f-strings technically breaks ST versions using Python 3.3. Hopefully that is not very many of them, but for the ones that have both runtimes, you can opt in to 3.8 with a `.python-version` file.